thoth-app/src/component/about.rs

```diff
@@ -0,0 +1,42 @@
+use yew::html;
+use yew::prelude::*;
+
+pub struct AboutComponent {}
+
+impl Component for AboutComponent {
+    type Message = ();
+    type Properties = ();
+
+    fn create(_ctx: &Context<Self>) -> Self {
+        AboutComponent {}
+    }
+
+    fn view(&self, _ctx: &Context<Self>) -> Html {
+        html! {
+            <div class="container">
+                <h1 class="title">{ "About Us" }</h1>
+                <div class="content">
+                    <p>{ "Thoth is an open metadata management and dissemination platform. As an organisation, we are focused on:" }</p>
+                    <ul>
+                        <li>{ "creation, curation, and dissemination of high-quality metadata records which are essential for the advancement of public knowledge;" }</li>
+                        <li>{ "promote openness in scholarly communications such as open access, open licensing, FLOSS, open data, open metadata, and open standards and protocols;" }</li>
+                        <li>{ "embrace infrastructural and platform diversity as an inherent component of a flourishing scholarly communications landscape;" }</li>
+                        <li>{ "provide high-quality solutions and services for metadata creation, management, dissemination, archiving and preservation." }</li>
```

Comment:
"promote"/"embrace"/"provide" should be "promoting"/"embracing"/"providing" for the grammar of the sentence to make sense ("we are focused on: [...] promoting")